### PR TITLE
Fix issues registration form

### PR DESCRIPTION
**Pull request by rnugraha**
- validate max 50 char for firstname & lastname
- validate max 150 char for some fields (dept, institution, function)
- added telephone validation
- placeholder text for other specialism

### CODE REVIEW

File: podium-gateway/src/main/webapp/app/account/register/register.component.html [hunk 1]
```diff
@@ -126,14 +126,17 @@ <h1 jhiTranslate="register.title">Registration</h1>
...
                     <input type="text" class="form-control" [(ngModel)]="registerAccount.telephone" id="telephone"
-                           #telephone="ngModel" name="telephone" required maxlength=100
+                           #telephone="ngModel" name="telephone" required maxlength=15 pattern="^[0-9]+$"
                            placeholder="{{'global.form.telephone.placeholder' | translate}}">
                     <div *ngIf="telephone.dirty && telephone.invalid">
                         <small class="form-text text-danger" *ngIf="telephone.errors.required">
                             Your telephone is required.
                         </small>
+                        <small class="form-text text-danger" *ngIf="telephone.errors.pattern">
+                            Your telephone must be numbers only.
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/account/register/register.component.html]: Your telephone number must be numbers only

File: podium-gateway/src/main/webapp/app/account/register/register.component.html [hunk 2]
```diff
@@ -126,14 +126,17 @@ <h1 jhiTranslate="register.title">Registration</h1>
                 <div class="form-group">
                     <label  class="form-control-label" for="telephone" >Telephone</label>
                     <input type="text" class="form-control" [(ngModel)]="registerAccount.telephone" id="telephone"
-                           #telephone="ngModel" name="telephone" required maxlength=100
+                           #telephone="ngModel" name="telephone" required maxlength=15 pattern="^[0-9]+$"
                            placeholder="{{'global.form.telephone.placeholder' | translate}}">
                     <div *ngIf="telephone.dirty && telephone.invalid">
                         <small class="form-text text-danger" *ngIf="telephone.errors.required">
                             Your telephone is required.
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/account/register/register.component.html]: Your telephone number is required.

File: podium-gateway/src/main/webapp/app/account/register/register.component.html [hunk 3]
```diff
@@ -181,9 +184,8 @@ <h1 jhiTranslate="register.title">Registration</h1>
                 <div class="form-group">
                     <label  class="form-control-label" for="jobTitle" >Function</label>
                     <input type="text" class="form-control" [(ngModel)]="registerAccount.jobTitle" id="jobTitle"
-                           #jobTitle="ngModel" name="jobTitle" required minlength="1"
-                           placeholder="{{'global.form.jobTitle.placeholder' | translate}}"
-                           pattern="^[_'.@A-Za-z]*$">
+                           #jobTitle="ngModel" name="jobTitle" required minlength="1" maxlength="150"
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/account/register/register.component.html]: Consistently apply attribute values with or without quotes. See line #129

Comment (rnugraha) [inline podium-gateway/src/main/webapp/app/account/register/register.component.html]: I propose that we are taking double quotes approach

Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/account/register/register.component.html]: @rnugraha I agree. More consistent with string values

**Pull request merged**